modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/RelationshipObjectFieldBusinessType.java

```diff
@@ -77,6 +95,67 @@ public Set<String> getRequiredObjectFieldSettingsNames() {
 				NAME_OBJECT_RELATIONSHIP_ERC_FIELD_NAME);
 	}
 
+	@Override
+	public Object getValue(ObjectField objectField, Map<String, ?> values)
```

Comment:
Why did you prefer Map<String, ?> than Map<String, Object> ?

Reply:
Because there was a place where I had `Map<String, Serializable>` instead of `Map<String, Object>`.

But now I changed it to `Map<String, Object>` since I can do `new HashMap<>(map)` when it is `Map<String, Serializable> map`

Thanks!